src/query/parser/promql/matchers.go

```diff
@@ -90,27 +90,31 @@ func NewAggregationOperator(expr *promql.AggregateExpr) (parser.Params, error) {
 		Without:      expr.Without,
 	}
 
-	op := getAggOpType(opType)
-	if op == common.UnknownOpType {
+	switch op := getAggOpType(opType); op {
```

Comment:
Nit: this `switch` has multi-line `case` blocks, would be a bit easier on the eyes if they were separated with empty lines.

Reply:
https://github.com/m3db/m3/pull/2780/commits/0897d9f9b8781d00e99f18cba704991c4ac45094#diff-fc8829d2c3cb3722ece0dc21d2fb4c8b2cf03230f5b641113b7413faf1660fa7R96-R118